modules/apps/frontend-data-set/frontend-data-set-taglib/src/main/resources/META-INF/resources/classic_display/init.jsp

```diff
@@ -25,7 +25,7 @@ page import="com.liferay.portal.kernel.util.PortalUtil" %>
 <liferay-theme:defineObjects />
 
 <%
-String randomNamespace = PortalUtil.generateRandomKey(request, "taglib_step_tracker") + StringPool.UNDERLINE;
+String randomNamespace = PortalUtil.generateRandomKey(request, "tablig_frontend_data_set_classic_display") + StringPool.UNDERLINE;
```

Comment:
Typo? Probably it should be taglib and not tablig :slightly_smiling_face:

Reply:
😱